src/foraging_gui/Foraging.py

```diff
@@ -155,6 +152,23 @@ def __init__(self, parent=None,box_number=1,start_bonsai_ide=True):
         self.to_check_drop_frames = 1 # 1, to check drop frames during saving data; 0, not to check drop frames
         self.session_run = False    # flag to indicate if session has been run or not
 
+        # Stage Widget
+        self.stage_widget = None
```

Comment:
@jessy-liao Our code automatically opens github issues for unique error strings, so if you just do:
```logging.error('stage widget failure <unique identifier for session/rig/mouse>')```

Then the error will open with the log attached, and all the information you requested. This will get you more information than asking a user to document things